pages/1.12/release-notes/1.12.3/index.md

```diff
@@ -0,0 +1,150 @@
+    - If `Anonymous Bind` is selected, the `JSON` sent to Bouncer `/ldap/config` API has no `lookup-dn` or `lookup-password` field.
+    - If `LDAP Credentials` is selected, use the current behavior. The `JSON` sent to Bouncer `/ldap/config` API has `lookup-dn` and `lookup-password` field. The `lookup-password` field may be an empty string.
+
+## Installation
+ - COPS-4282, DCOS_OSS-4613 - If you run the `dcos_generate_config` command with the `-validate` option, the command validates the configuration settings in your `config.yaml` file. In some cases, this option issued warning messages that validation failed for parameters that are no longer used. For example, some SSH parameters, such as `ssh_key_path` and `ssh_user`, have been deprecated. Previously, if you ran `dcos_generate_config` with the `-validate` option to check your configuration settings and these parameters were not specified, the command reported that the validation of configuration parameters had failed. With this release, the `--validate` option does not return validation failure messages for parameters that are no longer required for installation.
+
+- DCOS-15890 - The Preflight check on advance installer shows misleading information. This release improves an error message in case Docker is not running at the start of installation.
+
+## Marathon
+- COPS-3554 - The watcher loop process monitors and re-registers (if necessary) the Marathon leader after re-election.
+
+- COPS-3593, DCOS_OSS-4193 - In previous releases, you might have services that are managed by Marathon unable to restart if the container crashes or under certain DNS failure conditions. For example, restarting services might fail if the first ZooKeeper node or first DC/OS master is unreachable. Because this problem affects high availability for Marathon, a workaround (ping zk-1) was introduced for DC/OS 1.11.5 and 1.11.6 to address the issue. In this release, the underlying issue is resolved and you can safely remove the workaround if you have it deployed. For background information about the issue and the steps to remove the workaround, see [removing the patch for Marathon failing to start if the first DC/OS is not available](https://mesosphere-community.force.com/s/article/Critical-Issue-Marathon-MSPH-2018-0004).
+
+## Mesos
+- DCOS-46554 - This change forces Mesos master to have port resources in every offer unless the offer contains disks. This helps to reduce the number of offers with no ports which are not useful for most frameworks.
+
+## Metrics
+- DCOS-47991, DCOS_OSS-4760 - In the MWST 1.13, there are gaps in various Mesos Grafana dashboards. Further investigation revealed that metrics were occasionally missing the `framework_name=<framework_name>` tag but with an additional `role_name=<framework_name>` tag. This exercise revealed a bug in the Mesos input plugin in Telegraf that may be causing this issue. The solution is to fix the hashing function to avoid potentially mapping fields to wrong tags.
```

Comment:
Do we really want to mention MWST 1.13?   This is internal only testing, I don't think it adds much to the release notes to mention this.    How about this instead?   "An issue with the way the Mesos input plugin was mapping fields to tags for Telegraf, which led to gaps in Mesos Grafana dashboards, was corrected."